Client/SoundManager.cpp

```diff
@@ -28,6 +29,12 @@ void SoundManager::SM_StopSound(std::string soundName) {
 	StopSound(*sounds[soundName]);
 }
 
+void NCL::CSC8503::SoundManager::SM_PauseSound(std::string soundName)
```

Comment:
Can you get rid of the namespacing here please (I know its an auto generated thing but still)